libs/core/tests/unit_tests/utils/test_function_calling.py

```diff
@@ -71,6 +71,21 @@ def json_schema() -> Dict:
     }
 
 
+@pytest.fixture()
+def json_schema_lax() -> Dict:
```

Comment:
nit: a fixture probably doesn't make sense here too much abstraction for a unit test -- i'd just embed the test case into the function itself